DirectProgramming/DPC++/ParallelPatterns/histogram/src/main.cpp

```diff
@@ -0,0 +1,111 @@
+//==============================================================
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+#include <CL/sycl.hpp>
+#include "dpc_common.hpp"
```

Comment:
dpc_common.hpp should always be last in the list
Also need to add a comment in regards to location, should look like


// dpc_common.hpp can be found in the dev-utilities include folder.
// e.g., $ONEAPI_ROOT/dev-utilities/<version>/include/dpc_common.hpp
#include "dpc_common.hpp"